src/app/views/query-response/graph-toolkit/templates.json

```
@@ -0,0 +1,5 @@
+{
+    "/me/planner/tasks" : "https://mgt.dev/iframe.html?id=components-mgt-tasks--tasks",
```

Could you add a query paremeter to the url to identify the request as coming from the Graph Explorer?

ex: 

```suggestion
    "/me/planner/tasks" : "https://mgt.dev/iframe.html?id=components-mgt-tasks--tasks&source=ge",
```

---

Definitely. 

Though we've added the json file so that you can update the url easily.

---

@thewahome let's put the json file in the root directory so that it is easy to find.